packages/api-derive/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@polkadot/api-derive",
-  "version": "0.81.0-beta.7",
+  "version": "0.81.0",
```

Comment:
Why this change? Doing so will make the merge publish fail.

Reply:
i.e. it is now going to try and re-publich 0.81.0-beta.1 (over the previous already-published version), and fail.

Reply:
Yeah, right. It should be `0.82.0`. I just looked at @amaurymartiny PR from 2 months ago when he updated Metadata from v4 to v5 and I just copied his versioning mishap https://github.com/polkadot-js/api/pull/857/files :|

Reply:
Ok, the issue however is - we are not doing a stable release here. We have not tested this and any of the other changes sufficiently - so no bumps, we are not in a position to do this. Once we do, it goes out everywhere.,